### DISCUSSION TITLE
Azure Media Services - Doświadczenia z usługą/opinie/uwagi?

### DISCUSSION THREAD

**Discussion by phaate** [AzureCommunityPL/group#389] [Pomoc]
Hej!

Macie może jakieś doświadczenia z Azure Media Services? Znalazłem świetny wpis Furmana jednak... z 2017 roku.
http://cloudarchitects.pl/2017/07/wszystko-co-musisz-wiedziec-by-zaczac-programowac-azure-media-services/

Jak ta usługa zmieniła się od 2017 roku, macie może jakieś z nią nowsze doświadczenia lub małe porady na co zwrócić uwagę, jak się nie zabić i jak poprawnie się do tego zabrać? Robię małe rozeznanie do projektu :)

**Comment by mgrabarz** [comment]
Jakieś mam. Czego konkretnie potrzebujesz?

Możesz też podpytać @galante13, który pewnie siedzi w Media Services po uszy w SAG.

**Reply by galante13** [reply]
Niestety olaliśmy temat media services dawno temu, ale wraca do nas jak boomerang, lada moment zaczynam poca (dobry timing :)), także za jakiś czas będę już dużo na ten temat wiedział.

**Reply by phaate** [reply]
Na razie też tylko rozpoznaje temat. 
Nie do końca wszyscy się u nas lubią z .NETem więc ciekawi mnie jak teraz Azure radzi sobie z SDKami bo jeszcze do niedawna była opinia, że nie wszystkie biblioteki są "production ready" :P 

Generalnie dość standardowy projekt, security kamery + zapis transmisji + podgląd live + wykrywanie na żywo zdarzeń. 
To wszystko całkiem ładnie opisane jest w dokumentacji ale jest coś przez co poprzednie projekty się nie udały albo na co zwrócić konkretną uwagę, może jakieś materiały Ci się ostały/artykuły/poradniki, co zabiło projekt? @galante13

**Reply by smereczynski** [reply]
> Nie do końca wszyscy się u nas lubią z .NETem więc ciekawi mnie jak teraz Azure radzi sobie z SDKami bo jeszcze do niedawna była opinia, że nie wszystkie biblioteki są "production ready" :P 

Ale że jakie sdk nie są production ready? Czyje opinie? Bo brzmi to jak farmazon kogoś kto ich na oczy nie widział ;)

**Reply by phaate** [reply]
> 

Ja całe życie w .NET żyje więc sam nigdy tego nie sprawdzałem chociaż Microsoft teraz to "podobno" ładnie ustandaryzował - na podcascie .NET Rocks opowiadali jak teraz budują SDK, Api etc.

**Reply by galante13** [reply]
Projekt nie zabiło nic, nie użyliśmy po prostu media services, bo mamy bardzo specyficzny case (nasz ai wymaga sporo obróbki video, również nasz player jest mocno customowy). Teraz mamy inny temat, do którego będziemy używać najprawdopodobniej właśnie media services, dopiero zaczynam poca.

**Comment by smereczynski** [comment]
Jednym okiem śledzę bo raz czy dwa razy w roku temat wraca i raczej nie ma daleko idących zmian po 2017, jeśli nie rozpatrujemy indekserów. Wymyślić coś nowego w tym obszarze jest dość ciężko.

**Reply by phaate** [reply]
Czyli śmiało można opierać się też na starych materiałach - to dość dobra wiadomość, dzięki.
Masz może jakieś szczególne uwagi na co warto zwrócić szczególną uwagę przy dość standardowym projekcie, security kamery + zapis transmisji + podgląd live + wykrywanie na żywo zdarzeń?
